### PR TITLE
Most recent update on the Analyzer Software Requirements

### PR DESCRIPTION
**Pull request by AOrps**
- Updates on what is complete and tested and working 100%

### CODE REVIEW

File: Mentorship/Software Requirements - Analyzer.md [hunk 1]
```diff
@@ -71,42 +71,47 @@ stateDiagram-v2
...
+- [x] Manipulation on package name / version number
   - Package and Version Validation
 - :heavy_check_mark: Erroneous data mitgiation
   -  [x] One package per container
 
 
 ## Acceptance Criteria
-- [ ] The Analyzer pushse the final SARIF file to the Triage Portal's Endpoint
+- [x] The Analyzer pushse the final SARIF file to the Triage Portal's Endpoint
 - [ ] The Analyzer has multiple methods to pass the credentials to establish a connection to the Triage Portal
```
Comment (Cyber-JiuJiteria) [inline Mentorship/Software Requirements - Analyzer.md:104]: Anything that was not completed. Please more to a section `Future Improvements` with it's appropriate section (i.e., Acceptance Criteria, User Story, Security Requirement)

File: Mentorship/Software Requirements - Analyzer.md [hunk 2]
```diff
@@ -71,42 +71,47 @@ stateDiagram-v2
...
+  - [x] Remediation: Checksum validation
+- [x] Manipulation on package name / version number
   - Package and Version Validation
 - :heavy_check_mark: Erroneous data mitgiation
   -  [x] One package per container
 
 
 ## Acceptance Criteria
-- [ ] The Analyzer pushse the final SARIF file to the Triage Portal's Endpoint
+- [x] The Analyzer pushse the final SARIF file to the Triage Portal's Endpoint
```
Comment (glenda1015) [inline Mentorship/Software Requirements - Analyzer.md:103]: I think you meant to say "pushes"?

Comment (AOrps) [reply Mentorship/Software Requirements - Analyzer.md:105]: I think you are right lol

Comment (AOrps) [reply Mentorship/Software Requirements - Analyzer.md:105]: Good catch

**Review by Cyber-JiuJiteria — APPROVED** [review]

**Pull request merged**